go-controller/pkg/util/ovs.go

```diff
@@ -125,6 +125,67 @@ type execHelper struct {
 
 var runner *execHelper
 
+type ExecUtilRunSvc interface {
+	RunCmd(cmd kexec.Cmd, cmdPath string, envVars []string, args ...string) (string, string, error)
+	Run(cmdPath string, args ...string) (string, string, error)
+	RunWithEnvVars(cmdPath string, envVars []string, args ...string) (string, string, error)
+}
+
+// ExecUtilRunSvcImplStruct implements the methods defined in the ExecUtilRunSvc interface
+type ExecUtilRunSvcImplStruct struct {
```

Comment:
don't need to include the type in the name, so you can just drop the "Struct" an dmake it ExecUtilRunSvcImpl.

Reply:
Sure, I can  rename in the next commit once I receive more comments (if any) to the PR

Reply:
I would also name this to mirror the interface name and describe it's purpose
Assuming it's `ExecRunner`, is this `DefaultExecRunner`?

Reply:
I can drop the struct and per the suggestion above, I can rename it as ExecRunnerSvc.
@dave-tucker what is DefaultExecRunner? apologies, I did not get your question

Reply:
Sorry, I mean that `ExecRunnerSvc` doesn't make sense to me. It's not what i'd call a service (assuming that's what we're abbreviating to Svc) - it's simply something that implements the interface. So IMO, it's name should describe what type of implementation it is.
e.g, `io.Reader` is the interface, `io.PipeReader` reads from pipes.
Usually my default for `this is the only real implementation of this interface` is `Default$Interface` - which is where `DefaultExecRunner` is coming from.

Reply:
Anyway, naming is hard :man_shrugging:

Reply:
Renamed as `defaultExecRunner` per review suggestion in the latest PR